tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt

```diff
@@ -558,7 +540,7 @@ namespace FakeItEasy.Core
         public FakeCreationException(string message) { }
         public FakeCreationException(string message, System.Exception innerException) { }
     }
-    public class FakeManager : FakeItEasy.Core.IFakeCallProcessor

```

Comment:
Never noticed this before... `IFakeCallProcessor` is internal, so it shouldn't show up here. Well, apparently it's been fixed!